web/beacon-app/src/features/projects/hooks/useCreateProject.tsx

```diff
@@ -13,6 +15,7 @@ export function useCreateProject(): ProjectMutation {
       queryClient.invalidateQueries({ queryKey: [RQK.PROJECTS] });
       queryClient.invalidateQueries({ queryKey: [RQK.QUICK_VIEW] });
       queryClient.invalidateQueries({ queryKey: [RQK.PROJECT_QUICK_VIEW] });
+      toast.success(t`Success! You have created a new project.`);
```

Comment:
Ah, I see that you have implemented a Toast that triggers as soon as a request is successful. That's a great idea, but there may be situations where we need to have more control over when the Toast is displayed. For example, we might want to display it only after a few more instructions have been executed. In such cases, I would recommend adding it directly to the component which uses this hook to allow more granular control over when the Toast is shown.